common/workunit/workunit.cpp

```diff
@@ -7460,11 +7652,23 @@ IUserDescriptor *CLocalWorkUnit::queryUserDescriptor() const
     CriticalBlock block(crit);
     if (!userDesc)
     {
-        SCMStringBuffer token, user, password;
```

Comment:
I'm wondering if local workunits should have access tokens, or is that only for DALI workunits? (Derived class)